addons/osfstorage/listeners.py

```diff
@@ -1,16 +1,14 @@
 '''
 Listens for actions to be done to OSFstorage file nodes specifically.
 '''
-from modularodm import Q
-
 from website.project.signals import contributor_removed
 
 @contributor_removed.connect
 def checkin_files_by_user(node, user):
     ''' Listens to a contributor being removed to check in all of their files
     '''
     from addons.osfstorage.models import OsfStorageFileNode
-    files = OsfStorageFileNode.find(Q('node', 'eq', node) & Q('checkout', 'eq', user))
+    files = OsfStorageFileNode.objects.filter(node=node, checkout=user)
```

Comment:
You can use [`update()`](https://docs.djangoproject.com/en/1.11/ref/models/querysets/#update) here and eliminate the loop below. 

Just so you know, `update()` does bypass custom saves, but I checked with Wisecarver that it's perfectly fine to use here.